_data/authors.yml

```diff
@@ -4,7 +4,7 @@ peter_krauss:
   name: Peter Krauß
   web: http://www.upday.com
   email: peter.krauss@upday.com
-  bio: "1337?"
+  bio: "Peter joined upday at the very beginning."
```

Comment:
This sounds like a beginning of a story and makes me want to hear the rest. 
Could you use the bio and bio_long tags similar to what Maciej did for me?